src/com/amazon/ion/benchmark/IonSchemaUtilities.java

```diff
@@ -54,16 +55,20 @@ public class IonSchemaUtilities {
     /**
      * Check the validation of input ion schema file and will throw InvalidSchemaException message when an invalid schema definition is encountered.
      * @param inputFile represents the file path of the ion schema file.
-     * @throws Exception if an error occur when creating FileInputStream.
+     * @return schema loaded from input ISL file.
      */
-    public static void checkValidationOfSchema(String inputFile) throws Exception {
+    public static Schema checkValidationOfSchema(String inputFile) {
+        // Build ion schema system from input ISL file.
         IonSchemaSystem ISS = buildIonSchemaSystem(inputFile);
+        // Get the name of ISL file as schema ID.
         String schemaID = inputFile.substring(inputFile.lastIndexOf('/') + 1);
```

Comment:
Use a more robust way of extracting the filename in order to account for platform differences (`/` isn't the path separator on all platforms). e.g. `Paths.get(inputFile).toFile().getName()`

Reply:
> Use a more robust way of extracting the filename in order to account for platform differences (`/` isn't the path separator on all platforms). e.g. `Paths.get(inputFile).toFile().getName()`

Will update it in the next commit :)